addons/beehave/debug/tree_node.gd

```diff
@@ -95,25 +95,23 @@ func _calculate_initial_x(node: TreeNode) -> void:
 			node.x = node.get_previous_sibling().x + node.get_previous_sibling().item.size.x + sibling_distance
 		else:
 			node.x = 0
-	elif node.children.size() == 1:
-		var offset: float = (node.children.front().item.size.x - node.item.size.x) / 2
-		if node.is_most_left():
-			node.x = node.children.front().x + offset
-		else:
-			node.x = node.get_previous_sibling().x + node.item.size.x + sibling_distance
-			node.mod = node.x - node.children.front().x - offset
 	else:
-		var left_child = node.get_most_left_child()
-		var right_child = node.get_most_right_child()
-		var mid: float = (left_child.x + right_child.x + right_child.item.size.x - node.item.size.x) / 2
+		var mid: float
+		if node.children.size() == 1:
+			var offset: float = (node.children.front().item.size.x - node.item.size.x) / 2
+			mid = node.children.front().x + offset
+		else:
+			var left_child = node.get_most_left_child()
+			var right_child = node.get_most_right_child()
+			mid = (left_child.x + right_child.x + right_child.item.size.x - node.item.size.x) / 2
```

Comment:
No actual changes here just calculated `mid` in both situations instead of having `offset` and `mid`.